src/NuGet.Clients/NuGet.PackageManagement.VisualStudio/Utility/EnvDTEProjectUtility.cs

```diff
@@ -155,7 +155,7 @@ public static async Task<EnvDTE.ProjectItems> GetProjectItemsAsync(EnvDTE.Projec
             }
             if (createIfNotExists)
             {
-                // The JS Metro project system has a bug whereby calling AddFolder() to an existing folder that
+                // The JS Modern UI (Widnows 8 era) project system has a bug whereby calling AddFolder() to an existing folder that
```

Comment:
Widnows => Windows

Reply:
Can you make this JS Windows Store app like you did below?